### DISCUSSION TITLE
Will there will be fossify video player a full fledge video player

### DISCUSSION THREAD

**Discussion by Gamer750** [FossifyOrg/General-Discussion#410] [Ideas]
Will there will be fossify video player a full fledge video player obviously offline which support adding subtitle and etc

**Comment by bjlockie** [comment]
use VLC

**Comment by naveensingh** [comment]
I don't see the need for a separate app (yet). The Gallery's in-app video player will be improved with more features like gestures for brightness/volume control (edit: gestures are already there).
